pkg/client/client_test.go

```diff
@@ -211,24 +269,47 @@ var _ = Describe("Client", func() {
 			Expect(err).NotTo(HaveOccurred())
 			Expect(cl).NotTo(BeNil())
 
+			By("initially creating a Deployment")
 			dep, err := clientset.AppsV1().Deployments(ns).Create(dep)
 			Expect(err).NotTo(HaveOccurred())
 
-			By("updating the object")
+			By("updating the Deployment")
 			dep.Annotations = map[string]string{"foo": "bar"}
 			err = cl.Update(context.TODO(), dep)
 			Expect(err).NotTo(HaveOccurred())
 
-			By("writing the result back to the go struct")
+			By("validating updated Deployment exists")
 			actual, err := clientset.AppsV1().Deployments(ns).Get(dep.Name, metav1.GetOptions{})
 			Expect(err).NotTo(HaveOccurred())
 			Expect(actual).NotTo(BeNil())
 
 			close(done)
 		})
 
-		It("should update an existing new object from an unstructured object", func() {
-			// TODO(seans3): implement these
+		It("should update an existing new object from an unstructured object", func(done Done) {
```

Comment:
As I said earlier, not sure if `unstructured` tests help our usecase ?

Reply:
Phil asked to do the unstructured, which gets slightly better code coverage. So let's go with it for now.